test/DynamoCoreWpfTests/RecordedTests.cs

```diff
@@ -3827,6 +3829,1109 @@ public void Lacing_Deffect_5759()
 
         }
         #endregion
+
+        #region writing Recording Tests for Colin Zach training files
+
+        [Test]
+        public void MAGN_7348_Vignette_01_Plane_Offset()
+        {
+            preloadGeometry = true;
+
+            //Create planes
+            //Scenario
+            //1. Create one plane with plane.XY 
+            //2. Give positive length and width values and test the Rectangle.ByWidthHeight
+            //3. Create plane.Offset with heigth 2; test Rectangle.ByWidthHeight
+            //4. disconnect rectangle.BywidthHeight's width and length;
+            //5. reconnect rectangle.BywidthHeight's width and length;
+            //  http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7348
+
+            RunCommandsFromFile("MAGN_7348_Vignette_01_Plane_Offset.xml", (commandTag) =>
+            {
+                var workspace = ViewModel.Model.CurrentWorkspace;
+                if (commandTag == "FirstRun")
+                {
+                    Assert.AreEqual(5, workspace.Nodes.Count);
+                    Assert.AreEqual(3, workspace.Connectors.Count());       
+                    NodeModel node = ViewModel.Model.CurrentWorkspace.NodeFromWorkspace
+                        ("8042e93f-a000-49b0-8fc4-edcbc6fc767e");
+                    Assert.AreEqual(ElementState.Active, node.State);
+                                   
+                }
+                else if (commandTag == "SecondRun")
+                {
+                    var rectangle = GetPreviewValue("8042e93f-a000-49b0-8fc4-edcbc6fc767e");
+                    Assert.IsNotNull(rectangle);
+                    Assert.AreEqual(rectangle.ToString(), "Rectangle(Width = 12.000, Height = 15.000)");
+                }
+                else if (commandTag == "ThirdRun")
+                {
+                    var plane = GetPreviewValue("4d97bcf7-bb6d-4e47-b37e-d5e871bc1fd5");
+                    Assert.IsNotNull(plane);
+                    Assert.AreEqual(plane.ToString(), "Plane(Origin = Point(X = 0.000, Y = 0.000, Z = 2.000), Normal = Vector(X = 0.000, Y = 0.000, Z = 1.000, Length = 1.000), XAxis = Vector(X = 1.000, Y = 0.000, Z = 0.000, Length = 1.000), YAxis = Vector(X = 0.000, Y = 1.000, Z = 0.000, Length = 1.000))");
```

Comment:
Can you keep the line length not more than 100 characters, so that it will be redable during PR review.